### ISSUE TITLE
Auto-Analyzer on Demand for specific tests

### ISSUE THREAD

**Issue by dstewart-kb4** [reportportal/reportportal#2446]
**Describe the solution you'd like**
There is currently an endpoint to start auto-analyzer on demand for launches. It would be helpful to be able to specify which tests to auto-analyze, whether that's through another endpoint or via a filter on this existing one. The ability to run this while a launch is still in progress would be important as well.

**Is your feature request related to a problem? Please describe.**
This is related to https://github.com/reportportal/reportportal/issues/2440
If we were able to have control over when the individual test cases are auto-analyzed we could work around the problems in our framework we've discovered with Immediate Auto-Analyzer.

**Comment by Kozovskaya** [comment]
Hello @dstewart-kb4 
Are you aware about our new feature Immediate? Auto-Analysis https://reportportal.io/docs/analysis/ImmediateAutoAnalysis/
If you mark your test with special attribute, the analysis of the item will start upon its finish and you won't have to wait until the entire launch finishes. I believe it could meet your needs.

**Comment by dstewart-kb4** [comment]
Yes this feature request is actually coming up as a result of that. There are a couple issues my team and I are having while enabling immediate auto-analysis as I've reported here https://github.com/reportportal/reportportal/issues/2440
